packages/core/src/components/forms/_control-group.scss

```diff
@@ -273,4 +273,43 @@ Styleguide control-group
       border-radius: 0 0 $pt-border-radius $pt-border-radius;
     }
   }
+
+  /* fix for NumericInput inside ControlGroup */
```

Comment:
I think this implementation should match how we handle `.bp3-input` inside control group, see line 67.

the weird thing here is that NumericInput renders a ControlGroup, so we end up nesting control groups. this is typically not supported, but in this case I think it's worth fixing.

there's another issue here with z-indices, which we should also fix in this implementation:

<img width="290" alt="image" src="https://user-images.githubusercontent.com/723999/88196323-0c3b2580-cc0f-11ea-86e1-f8be7bf3c01e.png">


since this is a little more involved fix, I'll submit a PR with my proposed changes